[2021-12-18 16:11:51,547] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.git_push 2021-12-18T16:09:43.600276+00:00 [queued]>
[2021-12-18 16:11:51,554] {taskinstance.py:876} INFO - Dependencies all met for <TaskInstance: rank_update.git_push 2021-12-18T16:09:43.600276+00:00 [queued]>
[2021-12-18 16:11:51,555] {taskinstance.py:1067} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 16:11:51,555] {taskinstance.py:1068} INFO - Starting attempt 1 of 1
[2021-12-18 16:11:51,555] {taskinstance.py:1069} INFO - 
--------------------------------------------------------------------------------
[2021-12-18 16:11:51,569] {taskinstance.py:1087} INFO - Executing <Task(BashOperator): git_push> on 2021-12-18T16:09:43.600276+00:00
[2021-12-18 16:11:51,570] {standard_task_runner.py:52} INFO - Started process 4605 to run task
[2021-12-18 16:11:51,581] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'rank_update', 'git_push', '2021-12-18T16:09:43.600276+00:00', '--job-id', '143', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/rank_update.py', '--cfg-path', '/tmp/tmptg7h4i0g', '--error-file', '/tmp/tmpw19hnpf4']
[2021-12-18 16:11:51,581] {standard_task_runner.py:77} INFO - Job 143: Subtask git_push
[2021-12-18 16:11:51,632] {logging_mixin.py:104} INFO - Running <TaskInstance: rank_update.git_push 2021-12-18T16:09:43.600276+00:00 [running]> on host airflowvm
[2021-12-18 16:11:51,842] {taskinstance.py:1280} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=rank_update
AIRFLOW_CTX_TASK_ID=git_push
AIRFLOW_CTX_EXECUTION_DATE=2021-12-18T16:09:43.600276+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-12-18T16:09:43.600276+00:00
[2021-12-18 16:11:51,848] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-12-18 16:11:51,848] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'git --git-dir=/home/airflow/Test/.git/ --work-tree=/home/airflow/Test/.git/ push']
[2021-12-18 16:11:51,869] {subprocess.py:75} INFO - Output:
[2021-12-18 16:11:53,486] {subprocess.py:79} INFO - To https://github.com/nicework-jin/Test
[2021-12-18 16:11:53,486] {subprocess.py:79} INFO -    f731aaf..1693813  main -> main
[2021-12-18 16:11:53,490] {subprocess.py:83} INFO - Command exited with return code 0
[2021-12-18 16:11:53,522] {taskinstance.py:1184} INFO - Marking task as SUCCESS. dag_id=rank_update, task_id=git_push, execution_date=20211218T160943, start_date=20211218T161151, end_date=20211218T161153
[2021-12-18 16:11:53,547] {taskinstance.py:1245} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-12-18 16:11:53,574] {local_task_job.py:151} INFO - Task exited with return code 0
